inc/saiport.h

```diff
@@ -368,6 +368,14 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_QOS_SCHEDULER_GROUP_LIST,
 
+    /**
+     * @brief The maximum accumulative headroom size of a port in bytes
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_QOS_MAXIMUM_ACCUMULATIVE_HEADROOM_SIZE,
```

Comment:
1) As discussed, please add comments to say this is applicable only for per-port per-PG headroom model (i.e., SAI_BUFFER_POOL_ATTR_XOFF_SIZE = 0)

2) Consider renaming the attribute.
SAI_PORT_ATTR_BUFFER_MAX_HEADROOM_SIZE

Reply:
Fixed.